msal/src/main/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplication.kt

```diff
@@ -372,11 +373,6 @@ class NativeAuthPublicClientApplication(
                         }
                     }
                     is SignInCommandResult.CodeRequired -> {
-                        Logger.warn(
```

Comment:
We have this log here on purpose, please leave it in (notice how it's a warning and not an info). This is to deal with the scenario where a developer calls signIn(username, password) with an account that's an email + OTP signup. The password is silently ignored; this log tries to warn developers of that. Perhaps not the best solution, but best to leave it in for now.